### DISCUSSION TITLE
Provenance - remove or alter Child / Parent relationships.

### DISCUSSION THREAD

**Discussion by Bird-NZ** [ordinals/ord#3731] [Q&A] [answered]
Question:
With an Ordinal inscription, can I revoke or change a parent / child relationship?

Why?
I'm building out a collection of art on Bitcoin. 

1. I have an inscription that represents me an an artist
2. I create a collection inscription (either .txt of .html) and inscribe it with its parent (the inscription from step1)
3. I mint an .html leveraging recursion to assemble a number of small pieces of art (pre inscribed sections of a larger picture, think jigsaw pieces) to and make the collection it parent.

BUT there will be times were I want to remove or change the parent of a collection (remove collection) or assembled .html (remove art)

Hope this makes sense

**Comment by raphjaph** [comment, marked as answer]
Unfortunately this will not be possible. Like in real life you cannot change your biological parent.
